sdk/monitor/monitor-opentelemetry-exporter/src/Declarations/Constants.ts

```diff
@@ -1,8 +1,12 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+export enum ServiceApiVersion {
```

Comment:
@xirzec - quick check are we trying to phase out `enum`s for these kinds of values?

I don't think we'd hold this PR because this is just a preview, but perhaps in the near future...

Reply:
phase out in favor of what?

Reply:
enums are fine, the problem becomes when you want to support "extensible" enums. For those cases we're having the APIs in question take the actual literal type (usually string) and then providing a "KnownFooValues" type enum that gives hints about what is supported.

Reply:
Like if you wanted to support someone passing some new value here without re-releasing the SDK, you could make the signature take a string and this would become "KnownServiceVersions" or something.